src/core/Perspective.ts

```diff
@@ -35,19 +35,25 @@ export default class Perspective {
         this.#prologEngine = null
         this.#prologNeedsRebuild = true
 
-        this.#pubsub.subscribe(PubSub.LINK_ADDED_TOPIC, () => {
-            this.#prologNeedsRebuild = true
+        this.#pubsub.subscribe(PubSub.LINK_ADDED_TOPIC, (perspective: PerspectiveHandle) => {
+            if (perspective.uuid == this.uuid) {
+                this.#prologNeedsRebuild = true
+            }
         })
 
-        this.#pubsub.subscribe(PubSub.LINK_REMOVED_TOPIC, () => {
-            this.#prologNeedsRebuild = true
+        this.#pubsub.subscribe(PubSub.LINK_REMOVED_TOPIC, (perspective: PerspectiveHandle) => {
+            if (perspective.uuid == this.uuid) {
+                this.#prologNeedsRebuild = true
+            }
         })
 
         const that = this
 
         process.on("SIGINT", function () {
             that.#prologEngine?.close()
         });
+
+        this.renderLinksAdapter();
```

Comment:
Why is the return value not used here? `renderLinksAdapter()` also doesn't change the local perspective so is it just to initialize the link language in some way?

Reply:
Scrolling down I found the changes to `syncWithSharingAdapter()`. Of course, that function is responsible for getting the changes from the link language. So then this is just for initialization?